shell/platform/android/io/flutter/plugin/platform/PlatformPlugin.java

```diff
@@ -445,6 +445,10 @@ public void updateSystemUiOverlays() {
         }
       }
     }
+
+    if (currentTheme != null) {
+      setSystemChromeSystemUIOverlayStyle(currentTheme);
```

Comment:
can we add a test for this?

Reply:
Yes, good point

Reply:
@blasten I added one and I will add some additional tests for the `updateSystemUiOverlays` method in another PR.